### ISSUE TITLE
javascript error in code blocks - no more copy button, mangled output

### ISSUE THREAD

**Issue by ldemailly** [istio/istio.github.io#574]
In the console:
```
prism.js:21 Uncaught TypeError: Illegal constructor
    at n (prism.js:21)
    at prism.js:21
    at prism.js:20
    at Array.forEach (<anonymous>)
    at Prism.plugins.toolbar.hook (prism.js:20)
    at Object.run (prism.js:2)
    at Object.highlightElement (prism.js:2)
    at HTMLDocument.highlightAll (prism.js:2)
```

copy button is gone and further code blocks are gone:

![screen shot 2017-09-29 at 4 30 11 pm](https://user-images.githubusercontent.com/3664595/31039570-8b9bdce4-a533-11e7-8c83-7454af3f56de.png)

this happened gradually over some days (was working for me still while not working for others) but now it seems prevalent/broken everywhere (new version of prism is broken maybe?)

JS help needed kinda urgently (or an alternative for those code blocks)

cc @andraxylia who also stepped on this problem

**Comment by ldemailly** [comment]
Found https://github.com/PrismJS/prism/issues/1181

**Comment by ldemailly** [comment]
Seems to be (latest) chrome specific, I have version 61.0.3163.100 (Official Build) (64-bit)
on OS X 10.12.6

Safari still works

**Comment by ldemailly** [comment]
@geeknoid do you remember which selections you made on http://prismjs.com/download.html

coy maybe ? and which languages/plugins ?

seems like we need to add some clipboard script to get it to work (was hoping that maybe the latest version will have a fix)

**Comment by geeknoid** [comment]
I think we just need to load the clipboard.js script from a CDN in _layouts/base.html prior to the point where prism.js is loaded.

**Comment by ldemailly** [comment]
fixed in #579
